### PR TITLE
Install no sym link

### PR DESCRIPTION
**Pull request by solashRan**
The main purpose of the branch is adding the --no-link argument to install
when the arg is provided the install script will not create a sym link to manof at /usr/local/bin
so no need to run install with sudo.

### CODE REVIEW

File: install [hunk 1]
```diff
@@ -3,43 +3,62 @@
...
     parser.add_argument('-o', '--offline-install-path', help='The dir path to install pip packages from.')
+    parser.add_argument('--no-link', action='store_true',
+                        help='Do not remove or create symlink')
 
     return parser.parse_args()
 
 
-args = register_arguments()
+def _create_sym_link():
+    # symlink to run
```
Comment (omesser) [inline install:26]: newline

**Review by omesser — APPROVED** [review]
LGTM

File: install (hunk 1 above)
Comment (solashRan) [reply install:26]: done

**Review by omesser — APPROVED** [review]
Thanks!

**Pull request merged**